app/models/user.rb

```diff
@@ -57,7 +58,7 @@ def superadmin?
   def role_exists
     return if Permission.pluck(:role).include?(role)
 
-    errors.add(:role, 'doesnt_exist')
+    # errors.add(:role, 'doesnt_exist')
```

Comment:
Why do we commented this?

Reply:
Thats a mistake, i did when tested locally. good catch)

Reply:
Np)